runtime/doc/quickref.txt

```diff
@@ -919,7 +919,7 @@ Short explanation of each option:		*option-list*
 'spellsuggest'	  'sps'     method(s) used to suggest spelling corrections
 'splitbelow'	  'sb'	    new window from split is below the current one
 'splitright'	  'spr'     new window is put right of the current one
-'splitscroll'	  'spsc'    determines scroll behavior when splitting windows
+'splitscroll'	  'spsc'    determines scroll behavior for split windows
```

Comment:
This change was missed in https://github.com/vim/vim/commit/594f9e09cd68e6277b8aa08094405bc642c5792a.